sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -8598,7 +8582,22 @@
                 "json": ""
               },
               "customConditional": "",
-              "logic": [],
+              "logic": [
+                {
+                  "name": "Check Youth In Care Options",
+                  "trigger": {
+                    "type": "javascript",
+                    "javascript": "data.howWillYouBeAttendingTheProgram"
+                  },
+                  "actions": [
+                    {
+                      "name": "Update Youth In Care Options",
+                      "type": "customAction",
+                      "customAction": "var options = [\n  { label: 'Yes', value: 'yes' },\n  { label: 'No', value: 'no' }\n];\n\nif (data.howWillYouBeAttendingTheProgram === 'Part Time') {\n  options.push({ label: 'Prefer not to answer', value: 'preferNotToAnswer' });\n}\n\ncomponent.values = options;"
+                    }
+                  ]
+                }
+              ],
```

Comment:
Nice finding. Please confirm if the dry run submission validates the option values based on offering intensity. I believe it will. But if not then we must validate manually at API.

Reply:
Yeah changed the code to validate it. Thanks